src/components/tab-bar.component.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-unused-expressions */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Animated, LayoutAnimation } from 'react-native';
+import { TabBarBottom, NavigationActions } from 'react-navigation';
+
+import { animations } from 'config';
+
+const mapStateToProps = state => {
+  return {
+    isTabBarVisible: state.auth.isTabBarVisible,
+  };
+};
+
+class TabBarComponent extends Component {
+  props: {
```

Comment:
Sorry about asking this but what is the purpose of this?
I mean, I don't see any `@flow` on top of the file and I don't see this in `propTypes`